### DISCUSSION TITLE
多窗口打包找不到renderer下的资源

### DISCUSSION THREAD

**Discussion by AIrSkycc** [alex8088/electron-vite#305] [Q&A]
如题，官方文档并没有写明如何配置打包renderer下的所有资源，以及动态模块(dll, node, exe)的打包和导入

**Comment by alex8088** [comment]
renderer 资源正常引入即可，vite自动处理，也可以参考vite静态资源处理，对于dll, node, exe这些资源只适用于主进程，请参考 https://cn.electron-vite.org/guide/assets.html
